services/blockchain-indexer/shared/dataService/business/nft/supported.js

```diff
@@ -0,0 +1,53 @@
+/*
+ * LiskHQ/lisk-service
+ * Copyright © 2023 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+const { Logger } = require('lisk-service-framework');
+const { requestConnector } = require('../../../utils/request');
+const { getNetworkStatus } = require('../network');
+
+const logger = Logger();
+
+const PATTERN_SUPPORT_ALL_NFT_COLLECTIONS = '*';
```

Comment:
Constants should go in the constants file.

Reply:
Applied.